internal/cli/organizations.go

```diff
@@ -628,6 +646,10 @@ func listMembersRolesOrganizationCmd(cli *cli) *cobra.Command {
   auth0 orgs roles members ls <org-id> -r role -n 100
   auth0 orgs roles members ls <org-id> -r role -n 100 --json`,
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if inputs.Number < 1 || inputs.Number > 1000 {
```

Comment:
I imagine that you considered separating this into its own separate function. Was the idea here to keep the overhead low?

Reply:
> whether we could add a way to provide validation within a Flag struct rather than have the validation logic within the command logic

Good idea!

Reply:
Wdyt if we keep the current number validation in this PR to keep things simple, even tho repeated, and follow up with a PR to add a `Validation` field on the `Flag` struct?